HRIE sshogrin

sshogrin opened the pull request:
## Changes

- Add new icon for: HodorReflexesIconExtension

## Testing

Tested with:

```
/hodor.share test @kbernd
```

<img width="243" height="61" alt="Screenshot 2025-08-03 181650" src="https://github.com/user-attachments/assets/c4d8c40d-76aa-4536-9a25-048a3a7e3944" />

studleton commented:
Since there was a conflict with a new icon I was asked to add, I asked sshogrin to integrate his icons directly into the lib to simplify and avoid future conflicts.

m00nyONE commented:
There are some tests failing. Could you please have a look into that ? :-)

1. The file: icons/sshogrin/Trippypaw.dds is missing. https://github.com/m00nyONE/LibCustomIcons/actions/runs/16710711792/job/47309613598?pr=22
3. Duplicates found: https://github.com/m00nyONE/LibCustomIcons/actions/runs/16710711792/job/47309613638?pr=22 please remove the duplicate entries and/or adjust their icons

sshogrin commented:
I have deleted icons in the "sshogrin" folder, deleted duplicate icons directs in the LUA file, and made the name change to the "Trippypaw.dds" icon.
Not sure how that happened, I copied my original LUA file directly in here, and it was working perfectly before.
